src/openms/source/ANALYSIS/OPENSWATH/IonMobilityScoring.cpp

```diff
@@ -474,4 +648,189 @@ namespace OpenMS
     scores.im_xcorr_coelution_score = xcorr_coelution_score;
     scores.im_xcorr_shape_score = xcorr_shape_score;
   }
+
+  void IonMobilityScoring::driftIdScoring(const SpectrumSequence& spectra,
+                                          const std::vector<TransitionType> & transition,
+                                          MRMTransitionGroupType& trgr_detect,
+                                          OpenSwath_Scores &scores,
+                                          const double drift_target,
+                                          RangeMobility im_range,
+                                          const double dia_extract_window_,
+                                          const bool dia_extraction_ppm_,
+                                          const bool /* use_spline */,
```

Comment:
Good question, I was wondering that myself.. It's not needed, it's part of the other `driftScoring` functions as well, but I don't think they use it either. It might have been from the very initial implementation I guess. @jcharkow , do you know what the purpose of this was for in the initial `driftScoring` method /  if it's still used?

Reply:
I have removed the use_spline param from all the `driftScoring*` methods, because it doesn't seem to be used anywhere.